modules/ROOT/pages/getting-started-local-virt.adoc

```diff
@@ -0,0 +1,19 @@
+:page-partial:
+
+
+The https://gitlab.com/bootc-org/podman-bootc-cli[podman-bootc-cli] tool streamlines
+a local virtualization experience.
+
+This command will "self-install" the container into a disk image behind
+the scenes, and launch an interactive SSH session.
+
+```
+$ podman-bootc run quay.io/centos-bootc/centos-bootc:stream9
+```
+
+Other useful commands include:
+
+- `podman-bootc list`
+- `podman-bootc ssh`
+- `podman-bootc rm`
```

Comment:
I think having a tool for this is nice, but I feel like it hides important details. Some people will like this, some people want to know more and once they understand what is going on underneath will maybe choose their existing workflows, or to use the tool.

For the people that want to know more we should maybe give them a way to do this with [virt-install](https://docs.fedoraproject.org/en-US/fedora-coreos/provisioning-libvirt/) or [qemu](https://docs.fedoraproject.org/en-US/fedora-coreos/provisioning-qemu/)

Reply:
I see there is another site that has more 'Provisioning' pages on it:

https://docs-bootc-org-fedora-bootc-6570d042ee03afe91a435802a9746be5039.gitlab.io/fedora-bootc/qemu-and-libvirt/

Reply:
I'll mark this as resolved and comment on those pages.